README.md

```diff
@@ -301,3 +301,103 @@ Pretty stnadard, we simply collect all the hashes and the associated types for t
+
+Do the same after for the VSIX created using `nix build .#vscode-inferno-syntax-highlighting`.
+
+### Regular Inferno
+
+In a shell go to /inferno and run `nix build .#inferno-lsp-server`
+Run `ls -al result`
+Copy the nix/store ... directory to your clipboard. Open VScode, press ctrl + shift + P and search for Open User Settings. Search for Inferno,
+find the inferno LSP extension tab and open it. Paste the directory you copied into the `Path to the inferno-lsp-server executable` field. 
+
+Be sure to append `/bin/inferno-lsp-server` to the end of the directory, then restart VScode.
+
+Create a new file with the .inferno extension. If you begin typing an inferno command such as Array.argmax, the autocomplete box should appear.
+
+Next, if you have the inferno repo cloned into ~/inferno
+
+add
+
+```
+"tasks": [
+        {
+            "label": "inferno",
+            "type": "shell",
+            "command": "cd ~/inferno; nix run .#inferno -- ${file}",
+            "problemMatcher": [],
+            "group": {
+                "kind": "build",
+                "isDefault": true
+            }
+        }
+    ]
+```
+
+to your tasks.json folder in VScode. Otherwise you will need to change `cd ~/inferno;` to the directory where you stored the repo.
+
+You should now be able to build .inferno scripts using `ctrl + shift + B` in VScode
+
+Try saving the following into an .inferno file and compiling it.
+
+```
+let arr = [1, 2, 3, 4] in
+let sum = Array.sum arr in
+sum
+```
+
+The file should compile and output 
+
+`10.0`
+
+### Inferno-ML
+
+in a shell go to /inferno run `nix build .#packages.x86_64-linux.inferno-ml:exe:inferno-ml-lsp-server-ghc924`
+run `ls -al result`
+copy the nix/store ... directory to your clipboard. Open VScode, press `ctrl + shift + P` and search for `Open User Settings`. Search for Inferno,
+find the Inferno LSP extension tab and open it. Paste the directory you copied into the `Path to the inferno-lsp-server executable` field. 
+
+Be sure to append `/bin/inferno-ml-lsp-server` to the end of the directory. Then, restart VScode.
+
+Create a new file with the .inferno extension. If you begin typing an inferno command such as ML.stack, the autocomplete box should appear.
+
+If you have the inferno repo cloned into ~/inferno
+
+add
+```
+"tasks": [
+        {
+            "label": "inferno",
+            "type": "shell",
+            "command": "cd ~/inferno; nix run .#inferno-ml -- ${file}",
+            "problemMatcher": [],
+            "group": {
+                "kind": "build",
+                "isDefault": true
+            }
+        }
+    ]
+```
+to your tasks.json folder in VScode. Otherwise you will need to change "cd ~/inferno;" to the directory where you stored the repo.
+
+You should now be able to build .inferno scripts using `ctrl + shift + B` in VScode
```

Comment:
This section is almost identical to the other one. I think it might be better (more maintainable) to only include a single set of more general instructions. You could add some comments below the instructions for any differences between `inferno` and `inferno-ml` (e.g. using `nix develop .#inferno-ml` instead of `nix develop .#inferno`)